common/array.h

```diff
@@ -261,29 +270,34 @@ class Array {
 	}
 #endif
 
+    /** Return the size of the array. */
 	size_type size() const {
 		return _size;
 	}
 
+    /** Clear the array of all its elements. */
 	void clear() {
 		freeStorage(_storage, _size);
 		_storage = nullptr;
 		_size = 0;
 		_capacity = 0;
 	}
 
+    /** Erase the element at @p pos position. */
 	iterator erase(iterator pos) {
 		copy(pos + 1, _storage + _size, pos);
 		_size--;
 		// We also need to destroy the last object properly here.
 		_storage[_size].~T();
 		return pos;
 	}
-
+		
+    /** Check whether the array is empty. */
 	bool empty() const {
 		return (_size == 0);
 	}
 
+    /** Check if two arrays are of the same size. */
```

Comment:
It does not just check that the arrays have the same size, but also that they have the same content.
So in short: `Check if two arrays are identical`.